{"name": "test_delete_cache", "status": "failed", "statusDetails": {"message": "AssertionError", "trace": "self = <Test_ziyun.test_DNC.Test_device_ml object at 0x000001DA207BA128>\n\n    @allure.story(\"文件删除到回收站\")\n    def test_delete_cache(self):\n        rp_resp = requests.put(url + '/file/delete',\n                               json={'reason': '自动化删除', 'fileId': fileId}, headers={'token': token_data})\n        assertions.assert_code(rp_resp.status_code, 200)\n        rp_resp_json = rp_resp.json()\n>       assertions.assert_in_text(rp_resp_json['message'], 'Success')\n\ntest_DNC.py:189: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <Common.Assert.Assertions object at 0x000001DA1F5DA0F0>\nbody = 'Internal server error', expected_msg = 'Success'\n\n    def assert_in_text(self, body, expected_msg):\n        \"\"\"\n        验证response body中是否包含预期字符串\n        :param body:\n        :param expected_msg:\n        :return:\n        \"\"\"\n        try:\n            text = json.dumps(body, ensure_ascii=False)\n            # print(text)\n>           assert expected_msg in text\nE           AssertionError\n\n..\\Common\\Assert.py:59: AssertionError"}, "attachments": [{"name": "log", "source": "9f0f9559-fe08-41b7-aa7a-debe17c74a5d-attachment.txt", "type": "text/plain"}], "start": 1600413556768, "stop": 1600413557234, "uuid": "28c75d34-fa63-4d6c-9019-d3ff6dfc8e35", "historyId": "e391da405e1e1104c6fb5b8016d2a07f", "testCaseId": "a369cab7babed87b894dd7dcd3b5d46c", "fullName": "test_DNC.Test_device_ml#test_delete_cache", "labels": [{"name": "story", "value": "文件删除到回收站"}, {"name": "feature", "value": "设备目录分发页、回传页接口"}, {"name": "suite", "value": "test_DNC"}, {"name": "subSuite", "value": "Test_device_ml"}, {"name": "host", "value": "DESKTOP-FIOLAUQ"}, {"name": "thread", "value": "41740-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_DNC"}]}